test/TestAssets/performance/Perfy.TestAdapter/Perfy.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// This dll is a TestProject as well as a test adapter.We use it as a baseline for performance.
+// It does no actual discovery or run.It just returns as many results as we ask it to by TEST_COUNT env variable.
+// It has almost no overhead, so all the overhead we see is coming from TestPlatform.
+
+using System.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+
+namespace PerfyPassing
```

Comment:
Won't work. Our test assets don't use preview language features, yet.

Reply:
I know it won't work because I did it and it did not work. Same for Uri initializer. And I don't want to just add the lang feature tag into this project when the other projects don't use it.